environment/src/test/java/com/sequenceiq/environment/environment/validation/EnvironmentValidatorServiceTest.java

```diff
@@ -69,9 +76,15 @@
     @Mock
     private TagValidator tagValidator;
 
+    @Mock
+    private EntitlementService entitlementService;
+
     @InjectMocks
```

Comment:
The `@InjectMocks` annotation should be removed; it is superfluous since `underTest` will be always initialized explicitly in `initTests()`.